c2cgeoportal/lib/cacheversion.py

```diff
@@ -61,7 +61,8 @@ def __call__(self, request):
 
         response = self.handler(request)
 
-        response.headers["Access-Control-Allow-Origin"] = "*"
-        response.headers["Access-Control-Allow-Headers"] = "X-Requested-With, Content-Type"
+        if path[1] in CACHE_PATH:
```

Comment:
Here we fill this global value with the static view name (second path element) https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/__init__.py#L343-L351
Already used 7 lines above :-)